artemis-jms-client/src/main/java/org/apache/activemq/artemis/jms/client/compatible1X/ActiveMQTextCompabileMessage.java

```diff
@@ -51,4 +52,10 @@ public ActiveMQTextCompabileMessage(ClientMessage message, ClientSession session
    public ActiveMQTextCompabileMessage(TextMessage foreign, ClientSession session) throws JMSException {
       super(foreign, session);
    }
+
+
+   @Override
+   protected SimpleString checkPrefix(SimpleString address) {
+      return ActiveMQCompatibleMessage.checkPrefix1X(address);
```

Comment:
Should this be on all compatible1x messages

Reply:
this one was missing.. consider it a typo.. it's the actual issue

Reply:
Should the type of the class name itself be fixed, too? (ActiveMQText**Compabile**Message.java)

Reply:
E.g. why is ActiveMQTextCompabile special vs others like BytesMessage or ObjectMessage

Reply:
@graben ouch.. yeah.. I will fix it

Reply:
@michaelandrepearce I don't understand your question... all the other classes had checkPrefix added, only TextMessage was missing.

Reply:
That was my question. You answered it :)